### DISCUSSION TITLE
Bash script to install CloudStack

### DISCUSSION THREAD

**Discussion by siosios** [apache/cloudstack#9151] [Show and tell]
I have re-written a bash script for RHEL 9 based Linux distro's (please don't judge the coding, It's not my day job) to install cloudstack but can only test on a single node. it works pretty well for single node installations from my testing, feel free to check it out.

https://github.com/siosios/autoinstall_cloudstack

**Comment by yadvr** [comment]
Thanks for sharing! I'm trying to build this: https://c8k.in and I might steal something from these scripts :)

**Comment by siosios** [comment]
I have been working on this for a little while and updated the original script.

https://github.com/siosios/autoinstall_cloudstack/blob/dev/autoinstall_cloudstack-fixed.sh

it now has:
``` 
Install with rollback on failure and network restoration, network settings backup, input validation with retry, complete packages install with verification.
uninstall with network restoration, mysql backup, package removal, configuration files reset to stock.
and also Logging is integrated. 
```
*****This is still a work in progress but it does install ACS on EL9 minimal systems as expected******
